src/components/v2/appDetails/k8Resource/nodeDetail/NodeDetailTabs/terminal/Terminal.tsx

```diff
@@ -33,6 +34,7 @@ function TerminalView(terminalViewProps: TerminalViewProps) {
     const [firstMessageReceived, setFirstMessageReceived] = useState(false);
     const [popupText, setPopupText] = useState<boolean>(false);
     const isOnline = useOnline();
+    const [errorMessage, setErrorMessage] = useState<string>(undefined)
```

Comment:
Don't init states with undefined use blank for string type & null for an object.